### PR TITLE
Support added for sprint 5 us

### PR DESCRIPTION
**Pull request by swarudka**

### CODE REVIEW

File: gen/definitions/ipv4_access_list.yaml [hunk 1]
```diff
@@ -0,0 +1,509 @@
...
+  - yang_name: sequences/sequence
+    tf_name: sequences
+    type: List
+    attributes:
+      - yang_name: sequence-number
+        id: true
+        example: 11
+      - yang_name: remark
+        example: remark for access list
+        exclude_test: true
```
Comment (danischm) [inline gen/definitions/ipv4_access_list.yaml:17]: I think we include the remark as well.

File: gen/definitions/ipv6_access_list.yaml [hunk 2]
```diff
@@ -0,0 +1,432 @@
...
+  - yang_name: sequences/sequence
+    tf_name: sequences
+    type: List
+    attributes:
+      - yang_name: sequence-number
+        id: true
+        example: 22
+      - yang_name: remark
+        example: remark for access list
+        exclude_test: true
```
Comment (danischm) [inline gen/definitions/ipv6_access_list.yaml:17]: I think we include the remark as well.

**Review by danischm — CHANGES_REQUESTED** [review]

File: gen/definitions/ipv4_access_list.yaml (hunk 1 above)
Comment (swarudka) [reply gen/definitions/ipv4_access_list.yaml]: corrected

File: gen/definitions/ipv6_access_list.yaml (hunk 2 above)
Comment (swarudka) [reply gen/definitions/ipv6_access_list.yaml]: **corrected**

**Review by danischm — APPROVED** [review]

**Pull request merged**